numpy/linalg/lapack_lite/make_lite.py

```diff
@@ -23,7 +23,7 @@
 # Arguments to pass to f2c. You'll always want -A for ANSI C prototypes
 # Others of interest: -a to not make variables static by default
 #                     -C to check array subscripts
-F2C_ARGS = ['-A']
+F2C_ARGS = ['-A', '-Nx800']
```

Comment:
Something to do with function table sizes. It might no longer be needed now that dlapack is a little smaller